torch/csrc/jit/attributes.h

```diff
@@ -227,14 +227,27 @@ struct Attributes {
     out << "]";
   }
 
+  static void printStrList(
+      std::ostream& out,
+      const std::vector<std::string>& items) {
+    out << "[";
+    int i = 0;
+    for (auto& item : items) {
+      if (i++ > 0)
+        out << ", ";
+      out << "\"" << escapeString(item) << "\"";
+    }
+    out << "]";
+  }
+
   static std::string escapeString(std::string s) {
-    std::vector<char> search = {'\n', '\t', '\v'};
-    std::vector<std::string> replace = {"\\n", "\\t", "\\v"};
+    std::vector<char> search = {'\n', '\t', '\v', '\"'};
```

Comment:
We probably want to use https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/passes/python_print.cpp#L19 escapeString was never safe for parsing purposes, it was only trying to avoid printing newlines.

Reply:
While I'm at this, should I move this function to ATen?

Reply:
It's fine for this to stay in JIT.

Reply:
Is it ok to have a dependency on `passes/python_print`?